website/src/components/views/CreateJob.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import queryString from 'query-string'
+import { NavBar } from '../NavBar';
+
+class CreateJob extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onSubmitWithProps = this.onSubmitWithProps.bind(this)
+
+        const queryValues = queryString.parse(props.location.search)
+        let cronJob = {}
```

Comment:
Using `let` suggests to the reader that you intend to reassign a different value to this variable later on.
But you do not. So always try and use `const` which conveys the intent that you do not intend to reassign.

Reply:
so I can set the value of the const inside the if statement even if I define it as empty initially?